bmgui/parse_corpus_page.py

```diff
@@ -292,7 +294,11 @@ def _launch_parsing(self, master, corpus_year, database_type,
     def _corpus_parsing(rawdata_path, parsing_path, database_type):
         if not os.path.exists(parsing_path):
             os.mkdir(parsing_path)
-        parsing_dict, dic_failed = bp.biblio_parser(rawdata_path, database_type)
+        parsing_tup = bp.biblio_parser(rawdata_path, database_type,
+                                       inst_filter_list = None,
+                                       country_affiliations_file_path = institute_affil_file_path,
+                                       inst_types_file_path = inst_types_file_path)
```

Comment:
Those parameters are not recognized as functions arguments for the biblio_parser() method.

Reply:
This is related to the BiblioParsing new version as already mentionned.